src/averageAges.js

```diff
@@ -1,61 +1,86 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+// Methods for working with data base 'people'
+const methods = {
+  getFilteredPeopleBySex(people, sex) {
+    return people.filter(person => person.sex === sex);
+  },
```

Comment:
Instead of having separate filter methods, better create condition check methods.

```suggestion
  hasGender(person, sex) {
    return person.sex === sex;
  },
```